src/SDKs/Resource/Resource.Tests/Helpers/ManagementGroupsTestUtilities.cs

```diff
@@ -1,28 +0,0 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Net;
-using System.Text;
-using Microsoft.Azure.Management.ResourceManager;
-using Microsoft.Rest.ClientRuntime.Azure.TestFramework;
-using ResourceGroups.Tests;
-
-namespace Resource.Tests.Helpers
-{
-    public static class ManagementGroupsTestUtilities
-    {
```

Comment:
Tests can be skipped instead of deleted if this is temporary

Reply:
the related APIs were removed. I think we can remove the test cases and utilities. 

---
In reply to: [193487458](https://github.com/Azure/azure-sdk-for-net/pull/4418#discussion_r193487458) [](ancestors = 193487458)